mantidproject/mantid
# ISIS SANS: Add save other window

**Description of work.**

This PR adds save other functionality to the new GUI. This allows the three unique SANS save algorithms designed to interact with SASview to be easily run on workspaces currently in the ADS

**Report to:** sarah 

**To test:**

Code review
 * Get the SANS data attached to this PR #23664
 * Make sure the data is added to your directories path and then load MaskFile.txt as the user file and batch_mode_reduction.csv as the batch file.
 * Click process. 
 * Click on the save other button at the bottom of the GUI. 
 * Select a directory and attempt to save out some of the output workspaces (note trying to save out the unreduced workspaces will fail)
 * Put in a new filename and check that this is used if only one workspace is selected and appended if multiple are selected. 
 * Check that the ADS still functions as expected in mantidplot and the workbench.


Fixes #23746  




---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
